test/streamsearch.spec.js

```diff
@@ -1,9 +1,7 @@
-const { describe, it } = require('mocha');
```

Comment:
this import is not necessary, they are automatically available in global scope when executed via mocha runner